man/systemd-system.conf.xml

```diff
@@ -353,7 +353,7 @@
 
         <para>If <varname>RuntimeWatchdogSec=</varname> is set to a non-zero value, the watchdog hardware
         (<filename>/dev/watchdog0</filename> or the path specified with <varname>WatchdogDevice=</varname> or
-        the kernel option <varname>systemd.watchdog-device=</varname>) will be programmed to automatically
+        the kernel option <varname>systemd.watchdog_device=</varname>) will be programmed to automatically
```

Comment:
we actually are fine with either syntax, we make no distinction there, see proc_cmdline_key_streq() which hides these mistakes.

but the patch is right nonetheless, since in the docs we try to stick to use "_" everywhere, even if we accept "-"

Reply:
Thanks for pointing that out! When I saw the barebox patch today, I wondered why I didn't notice problems when I tested the barebox feature a few years ago.